pom.xml

```diff
@@ -162,6 +162,42 @@
 					<version>${jacoco.version}</version>
 				</plugin>
 
+				<!-- License management -->
+				<plugin>
+					<groupId>com.mycila</groupId>
+					<artifactId>license-maven-plugin</artifactId>
+					<version>4.1</version>
+					<configuration>
+						<properties>
+							<project.name>ecoCode Android plugin</project.name>
+							<project.description>Provides rules to reduce the environmental footprint of your Android applications</project.description>
+							<email>contact@ecocode.io</email>
+							<year>2020</year>
```

Comment:
Why do you want to set the license year to 2020 and not leave it as the default (the current year)?

Reply:
I never know if it is the creation year or the last publication year that must be set in the copyright. I do not know how it is important so I prefered to set the the creation year. However if you are sure that it is the current year that we must set, I can change the date.